os/mm/mm_heap/mm_check_heap_corruption.c

```diff
@@ -262,6 +262,6 @@ int mm_check_heap_corruption(struct mm_heap_s *heap)
 			return -1;
 		}
 	}
-
+	lldbg("No heap corruption detected\n");
```

Comment:
All other logs in this file use `mfdbg`. Why do you use `lldbg` here only?

Reply:
Yeah, It should be ```mfdbg``` only. I will change it.